unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -51,6 +97,8 @@ def complete(
         Returns:
             LanguageModelResponse: The LanguageModelResponse object.
         """
+        validate_required_parameters({"company_id": self.company_id})
```

Comment:
It looks to me that this is only used here. Why not in the other methods

Reply:
used in other methods as well